scripts/deployment/globals_gnosis_mainnet_qs_beta_expert4.json

```diff
@@ -0,0 +1,38 @@
+{
+  "contractVerification":true,
+  "useLedger":true,
+  "derivationPath":"m/44'/60'/2'/0/0",
+  "providerName":"gnosis",
+  "networkURL":"https://rpc.gnosischain.com",
+  "gasPriceInGwei":"4",
+  "gnosisSafeAddress":"0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
+  "gnosisSafeProxyFactoryAddress":"0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
+  "serviceRegistryAddress":"0x9338b5153AE39BB89f50468E608eD9d764B755fD",
+  "serviceRegistryTokenUtilityAddress":"0xa45E64d13A30a51b91ae0eb182e88a40e9b18eD8",
+  "olasAddress":"0xcE11e14225575945b8E6Dc0D4F2dD4C570f79d9f",
+  "multisigProxyHash130":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000",
+  "stakingNativeTokenAddress":"",
+  "stakingTokenAddress":"0xEa00be6690a871827fAfD705440D20dd75e67AB1",
+  "livenessRatio":"694444444444444",
+  "stakingActivityCheckerAddress":"",
+  "stakingFactoryAddress":"0xb0228CA253A88Bc8eb4ca70BCAC8f87b381f4700",
+  "stakingParams":
+    {
+      "metadataHash":"",
```

Comment:
in other places we added this ...

Reply:
This will be added when that field is filled with correct data during the deployment.

Reply:
Some of the data is filled retrospectively assuming the deployment is done via the UI.